<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>credit_manager</class>
 <widget class="QDialog" name="credit_manager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Credit Manager</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>521</width>
     <height>124</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="horizontalSpacing">
     <number>100</number>
    </property>
    <property name="verticalSpacing">
     <number>6</number>
    </property>
    <item row="1" column="0">
     <widget class="QLineEdit" name="charge_credit_input">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>Enter Amount To Deposit</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="redeem_coupon_label">
      <property name="text">
       <string>Redeem Coupon</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="redeem_coupon_input">
      <property name="placeholderText">
       <string>Enter Coupon Code</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="charge_credit_label">
      <property name="text">
       <string>Charge Account</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="redeem_coupon">
      <property name="text">
       <string>Redeem</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="charge_credit">
      <property name="text">
       <string>Charge</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>301</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>You Are Charging User:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="logged_in_user">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>logged_in_user</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
